### PR TITLE
chore: update types for appium/appium#18721

### PR DESCRIPTION
**Pull request by boneskull**
this won't pass until those updates are released and the deps are upgraded here

### CODE REVIEW

**Review by jlipps — APPROVED** [review]

**Review by mykola-mokhnach — APPROVED** [review]

File: test/unit/driver-specs.js [hunk 1]
```diff
@@ -100,6 +101,12 @@ describe('XCUITestDriver', function () {
       let device;
       let realDevice;
 
+      before(function () {
```
Comment (mykola-mokhnach) [inline test/unit/driver-specs.js:104]: I have fixed stubs, so this condition could be removed now

**Comment by boneskull** [comment]
this now depends on appium/appium#18817

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1768** <a href="https://app.graphite.dev/github/pr/appium/appium-xcuitest-driver/1768" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium-xcuitest-driver/1768?utm_source=stack-comment).

File: lib/commands/log.js [hunk 2]
```diff
@@ -204,7 +205,10 @@ export default {
         log.debug(closeMsg);
       });
     });
-    await /** @type {AppiumServer} */ (this.server).addWebSocketHandler(pathname, wss);
+    await /** @type {AppiumServer} */ (this.server).addWebSocketHandler(
+      pathname,
+      /** @type {WSServer} */ (wss)
```
Comment (jlipps) [inline lib/commands/log.js:210]: should we use a typedef to import WSServer since it's not actually used as a var?

**Review by jlipps — APPROVED** [review]

**Pull request merged**